### PR TITLE
Validate new experiments

### PR DESCRIPTION
**Pull request by jasonallen68**
# Problem space:
- Experiments are not being validated.
# Changes:
- [ ] Filter out invalid new experiments prior to sending the experiment list to the feature service.
- [ ] Changed version to 0.10.0.
# Should:
- [ ] Only send valid experiments, as defined in app.js, to the feature serivice.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -37,8 +38,7 @@ var _settings = exports._settings = {
 
   // This is a node-cron cron job. This will re-fetch the configuration on
   // a set interval, based on the cron-job configuration
-  cronJob: null,
-
+  cronJob: null
```
Comment (dncrews) [inline lib/index.js]: Any reason you're removing my valid commas?

File: lib/index.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ var _appConfig = exports._appConfig = {
   reference: null,
 
   // Object - Identifies the set of shared experiments
-  shared: { experiments: [] },
+  shared: { experiments: [] }
```
Comment (dncrews) [inline lib/index.js]: Any reason you're removing my valid commas?

File: lib/index.js (hunk 1 above)
Comment (jasonallen68) [inline lib/index.js]: The last property in an object doesn't need a trailing comma and it actually shows an error in an IDE like WebStorm.

Comment (dncrews) [inline lib/index.js]: It's only an error according to the ECMA 5 standard, so it's bad in .json files, but not in .js. I tend to leave them in all of my non-json-file objects, unless I have to support IE8. It makes subsequent additions to those objects have a cleaner diff. I would much rather leave these in. There's a setting you can set to have it not error out in WebStorm, I hope.

Comment (jasonallen68) [inline lib/index.js]: Okay. I won't remove them.  Would like me to undo those changes for this PR?

Comment (dncrews) [inline lib/index.js]: Preferably

**Pull request merged**